pgjdbc/src/main/java/com/yugabyte/ysql/YBjdbcHostnameVerifier.java

```diff
@@ -0,0 +1,225 @@
+package com.yugabyte.ysql;
+
+import org.postgresql.PGProperty;
+import org.postgresql.core.PGStream;
+import org.postgresql.jdbc.PgConnection;
+import org.postgresql.ssl.PGjdbcHostnameVerifier;
+import org.postgresql.util.GT;
+import org.postgresql.util.HostSpec;
+import org.postgresql.util.PSQLException;
+import org.postgresql.util.PSQLState;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLPeerUnverifiedException;
+import javax.net.ssl.SSLSession;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.security.cert.CertificateParsingException;
+import java.security.cert.X509Certificate;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static org.postgresql.Driver.*;
+import static org.postgresql.util.internal.Nullness.castNonNull;
+
+public class YBjdbcHostnameVerifier implements HostnameVerifier {
```

Comment:
Suggestion: YBjdbcHostnameVerifier -> YBManagedHostnameVerifier since this is a sample verifier for YB Managed cluster.